### ISSUE TITLE
Abnormal left ventricular endsystolic diameter

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7589]
**Preferred term label:**
Abnormal left ventricular endsystolic diameter

-- with increased and reduced terms

**Synonyms**
Abnormal left ventricular end-systolic dimension
Abnormal left ventricular end-systolic diameter

**Definition (free text, please give PubMed ID)**
A deviation from the normal range of the left ventricular endsystolic diameter. The measurement of the left ventricular endsystolic diameter is performed in the parasternal long-axis view, perpendicular to the long axis of the left ventricle at the tips of the mitral valve leaflets. It refers to the transverse (left-to-right) internal (luminal) distance, excluding thickness of walls. The normal range can be taken as 20-40 mm (total 2SD range 17.6–52.3 according to PMID: 24451180)

PMID: 24451180

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Reduced LVESD,  in 612124   PMID:12642359  5/8

**Comment by pnrobinson** [comment]
added
